privacyidea/static/components/config/views/config.smtp.html

```diff
@@ -10,9 +10,13 @@
                     ng-class="{ active: $state.includes('config.smtp.add')}"
                     ng-show="checkRight('smtpserver_write')">
                     <a ui-sref="config.smtp.add" translate>
-                        New SMTP server
-                    </a>
-                </li>
+                        New SMTP server</a></li>
+                <hr>
+                <li role="presentation">
+                    <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                       privacyideaVersionNumber+'/configuration/smtpserver_config.html#'}}">
```

Comment:
and here

Reply:
done in b5be0d720e77db9fe1eab889e9384eb1e4c7a521.